assets/js/state/sagas/databases.test.js

```diff
@@ -41,6 +43,53 @@ describe('SAP Systems sagas', () => {
     console.error.mockRestore();
   });
 
+  it('should update the absent_at field when the database instance is marked absent', async () => {
```

Comment:
Why this now in the top of the file?
I would put it at the end as it is at the end in the prod code as well

Reply:
done